app/views/katello/api/v2/subscriptions/show.json.rabl

```diff
@@ -59,10 +59,3 @@ node :activation_keys do |subscription|
     }
   end
 end
-
-node :host, :if => lambda { |sub| sub && sub.host } do |subscription|
```

Comment:
@thomasmckay Moving to AR would speed this up, I don't see why we can't make a subscription to host relation table and  use that. @jlsherrill thoughts? (I don't mean in this PR, just in general)